DOCS-2281: Install a Debian VM on Scale (#1076)
truenas/documentation: content/SCALE/Virtualization/CreatingManagingVMsSCALE.md

**Operating System values entered:**
* Guest Operating System: Linux
* Name: debianVM
* Description: Debian VM
* Click the Next Button.

**CPU and Memory values entered:**
* Change the memory size to 1024 MiB.
* Click the Next button.

**Disks values entered:**
* Select *Create new disk image*.
* Select the Zvol Location.
* Change the size to 30 GiB.
* Click the Next button.

**Network Interface values entered:**
* Attach NIC: Select the physical interface to associate with the VM.
* Click the Next button.

**Installation Media values entered:**
* In this case the installation iso was uploaded to /mnt/tank2/isostorage/. Click on the installation ISO, debian-11.0.0-amd64-netinst.iso. 
* Note: If the iso hadn't been uploaded previously, the user would select the "Upload an installer image file" box, select a dataset to upload the iso to, click the Choose file button, and click the Upload button. Wait for the upload to complete.
* Click the Next button.

**GPU values entered:**
* Click the Next button.

**Confirm Options:**
* Verify the information is correct and then click the Submit button.

{{< /expand >}}

> * Once the VM has been created, expand the VM by clicking on the down pointing arrow to the right of the new VM. Click the Start button.

Reviewer: ```suggestion
* Once the VM has been created, start it by expanding the VM entry (down pointing arrow to the right of the VM name) and click *Start*.
```

> * Click the Display button to step through the Debian installation.

Reviewer: ```suggestion
* Click *Display* to open a virtual monitor to the VM and see the Debian Graphical Installation screens.
```

{{< expand "Debian Install Example." "v" >}}

**Debian Graphical Install**

> * Press the Return key, on the keyboard, to start the Debian Graphical Install.

Reviewer: ```suggestion
* Press <kbd>Return</kbd> to start the Debian Graphical Install.
```
